pkgs/top-level/all-packages.nix

```diff
@@ -1689,6 +1689,8 @@ in
 
   nx-libs = callPackage ../tools/X11/nx-libs { };
 
+  httplz = callPackage ../tools/networking/httplz { };
```

Comment:
Put in the correct spot, alphabetically, please ;)

Reply:
Done